### DISCUSSION TITLE
怎么登录帐号？

### DISCUSSION THREAD

**Discussion by CoderTCY** [Harry-zklcdc/go-proxy-bingai#331] [Q&A] [answered]
如题，而且在cn.bing.com上找不到USER_KievRPSSecAuth这个cookie参数

**Comment by Harry-zklcdc** [comment]
退出重新登录

**Reply by Harry-zklcdc** [reply, marked as answer]
https://github.com/Harry-zklcdc/go-proxy-bingai/wiki/%E8%8E%B7%E5%8F%96-Cookie
